sim/warlock/pet_abilities.go

```diff
@@ -144,10 +144,23 @@ func (wp *WarlockPet) newLashOfPain() *core.Spell {
 }
 
 func (wp *WarlockPet) newShadowBite() *core.Spell {
-	baseCost := wp.BaseMana * 0.03
+	actionID := core.ActionID{SpellID: 54053}
+	baseCost := 131.0 // TODO: should be 3% of BaseMana, but it's unclear what that actually refers to with pets
+	critMultiplier := 1.5 + 0.1*float64(wp.owner.Talents.Ruin)
```

Comment:
Does it really benefit from Ruin? It isn't specified in the talent description, contrary to the Imp's Firebolt.